libs/ui/pagination/helm/src/lib/hlm-pagination-previous.component.ts

```diff
@@ -23,9 +25,7 @@ import { HlmPaginationLinkDirective } from './hlm-pagination-link.directive';
 	`,
 })
 export class HlmPaginationPreviousComponent {
-	public readonly class = input('');
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	public readonly link = input<string | any[] | null | undefined>();
-
-	protected _computedClass = computed(() => hlm('gap-1 pr-2.5', this.class()));
+	public readonly userClass = input<ClassValue>('', { alias: 'class' });
+	public readonly link = input<RouterLink['routerLink']>();
```

Comment:
Thanks, yes, I had considered this, but someone could alternatively attach a click event handler to the element instead of it being associated with a link in which case link would not be required, which is why I left it as optional.

Reply:
That makes sense, thanks!